src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -480,4 +490,64 @@ private String getInvoiceLineNumberEndpoint(String id) {
 
   }
 
+  /**
+   * Updates the invoice's poNumbers field, following an invoice line creation, update or removal.
+   * @param invoice - the invoice of the modified invoice line
+   * @param invoiceLine - the modified invoice line
+   * @param invoiceLineFromStorage - the old version of the invoice line
+   * @param requestContext - used to start new requests
+   */
+  private CompletableFuture<Void> updateInvoicePoNumbers(Invoice invoice, InvoiceLine invoiceLine,
+      InvoiceLine invoiceLineFromStorage, RequestContext requestContext) {
```

Comment:
Could you please extract methods where it possible to increase readability of the code